doc/changelog/04-tactics/19023-abstract-subproof.rst

```diff
@@ -0,0 +1,4 @@
+- **Added:**
+  :tacn:`abstract` support for proof blocks, e.g. `abstract:{ tac1. tac2. }`
```

Comment:
Observation: I suppose this won't work in Ltac* tactics because `{ ... }` is not supported, nor are periods allowed inside Ltac* tactics.  (Not suggesting that it should be supported there.)